### PR TITLE
sigrok homepages

### PR DESCRIPTION
**Pull request by A108384**
**Summary**
- Add homepage references to sigrok library builds:  libserialport, libsigrokdecode, libsigrok
- Add homepage to sigrok-cli
- Rebuild of pulseview since it has a couple dependencies on the libsigrok libraries

**Test Plan**
- Rebuild and install all packages 
- Successfully Execute Demo Device as part of pulseview.
- Verify expected libraries are loaded as part of pulseview overview
- Successfully execute multiple demo commands via sigrok-cli [wiki](https://sigrok.org/wiki/Sigrok-cli)

**Checklist**
- [X] Package was built and tested against unstable

### CODE REVIEW

**Comment by androidnisse** [comment]
Resolves https://github.com/getsolus/packages/issues/2888

File: packages/l/libsigrokdecode/package.yml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
     - https://sigrok.org/download/source/libsigrokdecode/libsigrokdecode-0.5.3.tar.gz : c50814aa6743cd8c4e88c84a0cdd8889d883c3be122289be90c63d7d67883fc0
+homepage   : https://sigrok.org
 license    : GPL-3.0-or-later
 component  : programming.library
 summary    : Provides the basic API for sigrok protocol decoders
 description: |
     libsigrokdecode is a shared library written in C which provides the basic API for running sigrok protocol decoders. The protocol decoders themselves are written in Python.
 builddeps  :
     - pkgconfig(python3)
+    - pkgconfig(python3-embed)
```
Comment (EbonJaeger) [inline packages/l/libsigrokdecode/package.yml:14]: This is already provided as a part of the `python3` package.

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
Could you add "Fixes https://github.com/getsolus/packages/issues/2888" to the `sigrock-cli` commit message, so the issue gets automatically closed? Thanks!

File: packages/l/libsigrokdecode/package.yml (hunk 1 above)
Comment (A108384) [reply packages/l/libsigrokdecode/package.yml]: Thanks. Please take another look now.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**